Misc/NEWS.d/next/Library/2020-03-16-03-03-21.bpo-38307.2cmw2i.rst

```diff
@@ -0,0 +1,4 @@
+Adds end line no in class' use: generating dependency.

```

Comment:
I can’t parse this line 🙂 
(and second line starting with colon seems strange)

Reply:
Neither could I nor the doc tests.  I replaced it.

Reply:
Thanks, @terryjreedy !
But how could you push a change to my forked repository?

(I am not objecting to it, I just do not understand how another person can push to mine repo)

Reply:
When you made the PR, you left a checkmark in the box that says to allow core developers ('Members of Python' when you mouse over the icons) to modify the files in this branch.  If you had removed it, I would have asked to put is back in if possible.  To pull to your local repository,  which you should do before making further changes, checkout this branch, endline-in-readmodule-module, and, I believe, if you defined 'origin' as the devguide suggests
```
git pull origin endline-in-readmodule-module
```
Don't use rebase.